internal/api/api_default.go

```diff
@@ -0,0 +1,131 @@
+/*
+ * Swagger API for Kubefirst - OpenAPI 3.0
+ *
+ * Sample API for Kubefirst UI - Local
+ *
+ * API version: 0.0.1
+ * Contact: kray@kubefrist.com
+ * Generated by: Swagger Codegen (https://github.com/swagger-api/swagger-codegen.git)
+ */
+package swagger
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/spf13/viper"
+	"net/http"
+)
+
+func ConfigsGet(w http.ResponseWriter, r *http.Request) {
+	fmt.Printf("Starting server at port 8081\n")
+	w.Header().Set("Content-Type", "application/json; charset=UTF-8")
+	w.Header().Set("Access-Control-Allow-Origin", "*")
+
+	w.WriteHeader(http.StatusOK)
+
+	configValues := []Config{
```

Comment:
we can do it later: the function is a little big, we can improve it together on a later point.

Reply:
Yeah. I thought the same. I'll definitely refactor this in a following PR